sdk/python/requirements.txt

```diff
@@ -85,7 +85,7 @@ requests==2.32.3
     #   requests-toolbelt
 requests-oauthlib==2.0.0
     # via kubernetes
-requests-toolbelt==0.10.1
+requests-toolbelt==1.0.0
```

Comment:
Did you execute the command `pip-compile --no-emit-index-url requirements.in` to update the requirements.txt?